### PR TITLE
Improve project maintenance and security

### PR DESCRIPTION
**Pull request by naveensrinivasan**
- Updated docs to reflect the correct criticality of checks.
- Fixes https://github.com/ossf/scorecard-webapp/issues/373
- Fixes https://github.com/ossf/scorecard-webapp/issues/374

### CODE REVIEW

File: scorecards-site/content/home.md [hunk 1]
```diff
@@ -231,37 +231,37 @@ You can learn more about the scoring criteria, risks, and remediation suggestion
...
 | [Binary Artifacts](https://github.com/ossf/scorecard/blob/main/docs/checks.md#binary-artifacts)    | Is the project free of checked-in binaries?                                                                                                           | High     |
-| [Branch Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection)  | Does the project avoid dangerous coding patterns in GitHub Actions?                                                                                   | Critical |
-| [Dangerous Workflow](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) | Does the project use [Branch Protection](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/about-protected-branches)? | High     |
+| [Branch Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection)  | Does the project avoid dangerous coding patterns in GitHub Actions?                                                                                   | High     |
```
Comment (gabibguti) [inline scorecards-site/content/home.md:256]:
```suggestion
| [Branch Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection)  | Does the project use [Branch Protection](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/about-protected-branches)?                                                                                   | High     |
```

File: scorecards-site/content/home.md [hunk 2]
```diff
@@ -231,37 +231,37 @@ You can learn more about the scoring criteria, risks, and remediation suggestion
...
+| [Dangerous Workflow](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) | Does the project use [Branch Protection](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/about-protected-branches)? | Critical |
```
Comment (gabibguti) [inline scorecards-site/content/home.md:257]:
```suggestion
| [Dangerous Workflow](https://github.com/ossf/scorecard/blob/main/docs/checks.md#dangerous-workflow) | Does the project avoid dangerous coding patterns in GitHub Actions? | Critical |
```

File: scorecards-site/content/home.md [hunk 3]
```diff
@@ -231,37 +231,37 @@ You can learn more about the scoring criteria, risks, and remediation suggestion
...
 | [Code Review](https://github.com/ossf/scorecard/blob/main/docs/checks.md#code-review)              | Does the project require code review before code is merged?                                                                                           | High     |
 | [Contributors](https://github.com/ossf/scorecard/blob/main/docs/checks.md#contributors)            | Does the project have contributors from at least two different organizations?                                                                         | Low      |
```
Comment (gabibguti) [inline scorecards-site/content/home.md:259]:
```suggestion
| [Contributors](https://github.com/ossf/scorecard/blob/main/docs/checks.md#contributors)            | Does the project have contributors from at least three different companies?                                                                         | Low      |
```

**Comment by naveensrinivasan** [comment]
@gabibguti Thank you!

**Review by gabibguti — APPROVED** [review]

**Review by spencerschrock — APPROVED** [review]

File: scorecards-site/content/home.md [hunk 4]
```diff
@@ -231,37 +231,37 @@ You can learn more about the scoring criteria, risks, and remediation suggestion
...
+| [Dangerous Workflow](https://github.com/ossf/scorecard/blob/main/docs/checks.md#dangerous-workflow) | Does the project avoid dangerous coding patterns in GitHub Actions? | Critical |
 | [Code Review](https://github.com/ossf/scorecard/blob/main/docs/checks.md#code-review)              | Does the project require code review before code is merged?                                                                                           | High     |
-| [Contributors](https://github.com/ossf/scorecard/blob/main/docs/checks.md#contributors)            | Does the project have contributors from at least two different organizations?                                                                         | Low      |
+| [Contributors](https://github.com/ossf/scorecard/blob/main/docs/checks.md#contributors)            | Does the project have contributors from at least three different companies?                                                                         | Low      |
```
[the remark this answers is not in the document]
Comment (naveensrinivasan) [reply scorecards-site/content/home.md:259]: I can do that. Thanks for the review!

[the remark this answers is not in the document]
Comment (naveensrinivasan) [reply scorecards-site/content/home.md]: 👍

**Pull request merged**